app/javascripts/tiles/downloads.jsx

```diff
@@ -378,7 +400,9 @@ var DownloadsTile = React.createClass({
       <div className="tile">
         <div className="tile is-parent is-8">
           <div className={'tile is-child notification ' + this.props.colorclassName}>
-            {chart}
+            <div>
```

Comment:
I _think_ this extra div container will prevent the infinite growth bug. weird but maybe worth a shot!